roles/yarn/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+npm_prefix: "/usr/local/lib/npm"
+
+node_version: 14.x
+yarn_version: 1.22.4
```

Comment:
I wonder where's the best place to have these versions. These are equivalent to the bundler version. How does ofn-install know what bundler version to install?

Reply:
For things that are very specific to a role, we would normally put them in the role's defaults. For global configs like this I think we would often put them in `inventory/group_vars/all.yml`. 

`all.yml` is a bit of a monster, and it would be nice to tidy it up at some point, but not in this PR.

The `Bundler` version is a special case, we dynamically pull it from the `Gemfile.lock` during deployment, in case it changes in the main codebase.

Reply:
ok, I rebased the PR and added one new commit to move these variable to all.yml.